src/Models/Environment.php

```diff
@@ -900,11 +900,14 @@ public function wake()
         $domains = array_filter(
             $this->getDomains()->all(),
             function ($domain) {
-                return !empty($domain->get('dns_zone_name'));
+                $domain_type = $domain->get('type');
+                if (empty($domain_type) || "platform" != $domain_type)
+                    return false;
+                return true;
             }
         );
         $domain = array_pop($domains);
-        $response = $this->request()->request("http://{$domain->id}/pantheon_healthcheck");
+        $response = $this->request()->request("https://{$domain->id}/pantheon_healthcheck");
```

Comment:
Does guzzle follow redirects? Test on the live env of a site that redirects via php in settings.php

Reply:
This URL doesn't redirect.  It just serves an OK.
https://live-ryan-marks.pantheonsite.io/pantheon_healthcheck

Reply:
Right, I forgot that the prepend is going to grab this before it gets to the Drupal router. LGTM.